### ISSUE TITLE
[CoE Starter Kit - BUG] ISSUE TITLE

### ISSUE THREAD

**Issue by JohnStraumannMSFT** [microsoft/coe-starter-kit#5924]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

FYI I got an "out of date" message in the COE today, and when I click the link it's broken (see attachments).

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

1.7

### What app or flow are you having the issue with?

All

### What method are you using to get inventory and telemetry?

None

### Anything else?

![COE1](https://github.com/microsoft/coe-starter-kit/assets/95629603/f3892211-2115-4783-9e7e-4ea94f69532a)
![COE2](https://github.com/microsoft/coe-starter-kit/assets/95629603/2852d4b1-09b8-40aa-954b-4164e3a9e353)

**Comment by Jenefer-Monroe** [comment]
Thank you for posting, I dont htink this feature made it to the new Command Center. Will look at this.

**Comment by manuelap-msft** [comment]
You're right, we didn't add this feature to the new version of the Command Center. Updated for July release
![image](https://github.com/microsoft/coe-starter-kit/assets/55705669/1aa0db96-3787-488e-b8e6-cbc8b3228eb1)

![image](https://github.com/microsoft/coe-starter-kit/assets/55705669/d74779bc-dbfe-440b-b393-4362f8c525f5)
